tests/setup.test.js

```diff
@@ -42,11 +42,10 @@ test('getChallenge for challengeId = 2', function (t) {
 
     return server.inject({ method: 'GET', url: '/getChallenge' },
       function (res) {
-        var expected = '[{'
-          + '"chal_id":2,"chal_title":"Challenge Number 2",'
-          + '"chal_desc":"How can I...?","tags_name":"Corporate",'
-          + '"org_id":1,"tags_id":2'
-          + '}]';
+        var expected = '{"chal_id":2,"chal_title":"Challenge Number 2",'
+          + '"chal_desc":"How can I...?","tags_name":"Corporate","org_id":1,'
+          + '"tags":[{"id":2,"name":"Corporate"}]}';
+
         t.equal(res.payload, expected, 'server is up and running!');
```

Comment:
Maybe a better comment for this test?

Also it may be easier to read if we use res.result and deepEqual it? I don't know if there is any disadvantage doing it this way though.